sample/src/main/java/com/github/pedrovgs/sample/LynxApplication.java

```diff
@@ -30,6 +31,8 @@
   @Override public void onCreate() {
     super.onCreate();
     LynxShakeDetector lynxShakeDetector = new LynxShakeDetector(this);
+    LynxConfig lynxConfig = new LynxConfig();
+    lynxConfig.setMaxNumberOfTracesToShow(4000).setFilter("LynxFilter");
     lynxShakeDetector.init();
```

Comment:
Thanks for the sample update :) But don't we forget to pass the new ``lynxConfig`` instance to the ``lynxShakeDetector.init()`` call?